src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx

```diff
@@ -33,7 +33,6 @@ import {
   StyledList,
   Table,
   TdStyle,
-  Th,
```

Comment:
If this `Th` isn't used anymore, can we remove it from `styles.ts` as well?

Reply:
I removed the export. It is used inside the styles file.